sql/my_json_writer.cc

```diff
@@ -217,7 +217,24 @@ void Json_writer::add_size(longlong val)
 void Json_writer::add_double(double val)
 {
   char buf[64];
-  size_t len= my_snprintf(buf, sizeof(buf), "%lg", val);
+  size_t len;
```

Comment:
Would it be easier, and also avoid edge cases when `val` is beyond than what's defined in `my_json_writer.h`, by using the following:
```
size_t len= my_snprintf(buf, sizeof(buf), "%.*lg",
          2, val);
```
If this works, we also avoid adding additional CONSTANT definitions in the header file.